prime-router/src/main/kotlin/fhirengine/engine/FHIRConverter.kt

```
@@ -232,7 +234,14 @@ class FHIRConverter(
                             actionHistory.trackCreatedReport(routeEvent, report, blobInfo = blobInfo)
 
                             val bundleDigestExtractor = BundleDigestExtractor(
-                                FhirPathBundleDigestLabResultExtractorStrategy()
+                                FhirPathBundleDigestLabResultExtractorStrategy(
+                                    CustomContext(
```

Without a custom context calls to `.resolve()` fail silently